File: king-admin-java/src/main/java/com/oukingtim/domain/TbDict.java
package com.oukingtim.domain;

import com.baomidou.mybatisplus.annotations.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * Created by oukingtim
 */
@TableName("tb_dict")
@Data
@EqualsAndHashCode(callSuper = false)
public class TbDict extends BaseModel<TbDict> {

    private String code;
    private String text;
    private String remark;
    private String dictClassId;

}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/domain/TbDict.kt
package com.oukingtim.domain

import com.baomidou.mybatisplus.annotations.TableName

/**
 * Created by oukingtim
 */
@TableName("tb_dict")
class TbDict : BaseModel<TbDict>() {
    val code: String? = null
    val text: String? = null
    val remark: String? = null
    val dictClassId: String? = null
}